plugins/diagnose.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: diagnose
+spec:
+  version: v0.1.1
+  homepage: https://github.com/alecjacobs5401/kubectl-diagnose
+  shortDescription: Find and debug Kubernetes Pods that are "Not Ready"
```

Comment:
no need to say "Kubernetes", in kubectl context, Pod is specific enough.

Reply:
🤦 right lol. Thanks!